### ISSUE TITLE
Create Docker image for application

### ISSUE THREAD

**Issue by dzieciou** [kodujdlapolski/tumieszkaj#2]
Include

- postgresql
- conda
- application with crawler

**Comment by ad-m** [comment]
I am not sure that put PostgreSQL & crawler in one Docker image is good idea.

**Comment by dzieciou** [comment]
@ad-m Right. That's definitely not a good idea if we want to scale crawler instances (Scrapy) and application instances separately from DB.

We will give a better thought to application infrastructure, once we know more what we want to build.
